### PR TITLE
Add Managed-Tenant-Quota resources

### PR DESCRIPTION
**Pull request by dshchedr**
##### Short description:
Add OCP resources: MTQ and ResourceQuota and VirtualMachineMigrationResourceQuota

### CODE REVIEW

File: ocp_resources/resource.py [hunk 1]
```diff
@@ -297,6 +297,7 @@ class ApiGroup:
         V2V_KUBEVIRT_IO = "v2v.kubevirt.io"
         VELERO_IO = "velero.io"
         VM_KUBEVIRT_IO = "vm.kubevirt.io"
+        MTQ_KUBEVIRT_IO = "mtq.kubevirt.io"
```
Comment (omrirh) [inline ocp_resources/resource.py:300]: should be added in alphabetic order

File: ocp_resources/mtq.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from ocp_resources.resource import Resource
+
+
+class MTQ(Resource):
+    """
+    MTQ object.
```
Comment (omrirh) [inline ocp_resources/mtq.py:6]: please add reference to mtq CRD

**Review by rnetser — DISMISSED** [review]

**Review by rnetser — APPROVED** [review]

**Review by omrirh — APPROVED** [review]

**Pull request merged**